### PR TITLE
Remove completed TODO bugs.

### PR DESCRIPTION
**Pull request by sachinprasadhs**
Removed completed TODO bugs to avoid confusion, which is marked as fixed internally.

### CODE REVIEW

**Review by sirakiin — APPROVED** [review]

File: tensorflow/lite/BUILD [hunk 1]
```diff
@@ -1025,7 +1024,7 @@ cc_test(
         "testdata/multi_add.bin",
     ],
     tags = [
-        "no_windows",  # TODO(b/194459105): the test is flaky.
+        "no_windows",
```
Comment (mihaimaruseac) [inline tensorflow/lite/BUILD:1027]:
```suggestion
```

If the bug is completed, enable the test again

File: tensorflow/lite/BUILD [hunk 2]
```diff
@@ -1090,7 +1089,7 @@ cc_test(
     srcs = ["optional_debug_tools_test.cc"],
     data = ["testdata/add.bin"],
     tags = [
-        "nomsan",  # TODO(b/186359792)
+        "nomsan",
```
Comment (mihaimaruseac) [inline tensorflow/lite/BUILD:1092]:
```suggestion
```

If bug is fix, remove tag

File: tensorflow/lite/core/interpreter.cc [hunk 3]
```diff
@@ -49,8 +49,6 @@ limitations under the License.
 #endif
 #endif  // defined(__APPLE__)
 
-// TODO(b/132087118): move static_assert to c_api_internal when compiled with
-// C++.
 static_assert(sizeof(TfLiteFloat16) == sizeof(uint16_t),
```
Comment (mihaimaruseac) [inline tensorflow/lite/core/interpreter.cc:52]: Please keep this one. The need to move to this is still there, the bug was closed due to bankruptcy.

**Review by mihaimaruseac — CHANGES_REQUESTED** [review]

File: tensorflow/lite/BUILD [hunk 4]
```diff
@@ -1090,7 +1088,6 @@ cc_test(
     srcs = ["optional_debug_tools_test.cc"],
     data = ["testdata/add.bin"],
     tags = [
```
Comment (mihaimaruseac) [inline tensorflow/lite/BUILD:1090]:
```suggestion
```

File: tensorflow/lite/BUILD [hunk 5]
```diff
@@ -1090,7 +1088,6 @@ cc_test(
...
-        "nomsan",  # TODO(b/186359792)
     ],
```
Comment (mihaimaruseac) [inline tensorflow/lite/BUILD:1091]:
```suggestion
```

**Review by mihaimaruseac — CHANGES_REQUESTED** [review]

**Review by mihaimaruseac — APPROVED** [review]

**Pull request merged**